### PR TITLE
Add Huisman-2019-110

### PR DESCRIPTION
**Pull request by mathildavz**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

### CODE REVIEW

**Comment by mathildavz** [comment]
The `concepticon test` was passed:
INFO    all integrity tests passed: OK

**Comment by chrzyki** [comment]
Hey,

thank you very much for preparing this list and the PR. I'd very much like to review this list but I'm still on holiday until August 6th. Could the PR wait until then and I'll do the review immediately on the 7th? Thanks!

**Comment by AnnikaTjuka** [comment]
@chrzyki, sure! I'll moderate the PR.

File: concepticondata/conceptlists/Huisman-2019-110.tsv [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Huisman-2019-110-100	100	black1	black	黒	kuro	noun																
+Huisman-2019-110-101	101	black2	black	黒い	kuroi	adjective	163	BLACK														
+Huisman-2019-110-102	102	night	night	夜	yoru		1233	NIGHT														
+Huisman-2019-110-103	103	hot	hot	暑い	atsui		1286	HOT														
+Huisman-2019-110-104	104	cold	cold	寒い	samui		1287	COLD														
+Huisman-2019-110-105	105	full	full	いっぱい	ippai		1429	FULL														
+Huisman-2019-110-106	106	new	new	新しい	atarashii		1231	NEW														
+Huisman-2019-110-107	107	good	good	良い	yoi		1035	GOOD														
+Huisman-2019-110-108	108	round	round	丸い	marui		1395	ROUND														
+Huisman-2019-110-109	109	to dry	to dry	乾く	kawaku		1398	DRY
```
Comment (chrzyki) [inline concepticondata/conceptlists/Huisman-2019-110.tsv:110]: Not sure here, but I don't think its the property but rather the action of 'becoming dry', e.g. `1786/DRY UP`?

File: concepticondata/conceptlists/Huisman-2019-110.tsv-metadata.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "ROMANISATION"
+                    },
+                    {
+                        "datatype": "string",
+                        "name": "NOTES"
+                    },
+                    {
```
Comment (chrzyki) [inline concepticondata/conceptlists/Huisman-2019-110.tsv-metadata.json:62]: There's multiple empty columns after the `CONCEPTICON_GLOSS` row. Ideally these would be removed and the empty column definition removed here.

**Review by chrzyki — CHANGES_REQUESTED** [review]
Thanks for preparing this list! First round of small reviews/comments. My knowledge of Japanese is very limited and my suggestions might off, so take with a grain of salt. :)

**Comment by mathildavz** [comment]
@chrzyki Thank you for fixing the issues with the checks! Is there anything left to do or can this be merged?

**Comment by chrzyki** [comment]
Not from me, thank you very much again for preparing this list!

**Review by chrzyki — APPROVED** [review]

**Pull request merged**